packages/frontend/scss/message/_message-attachment.scss

```diff
@@ -79,7 +79,18 @@
     border-bottom-right-radius: 0px;
   }
 
-  &.content-above {
+  // "Has content above" selector
+  // (i.e. author title, "Forwarded" indicator, quote).
+  // FYI there are duplicates of this code.
```

Comment:
Wouldn't that be a reason to use a mixin here?

```
// Mixin to apply styles when an element has visible preceding content
// (i.e. author title, "Forwarded" indicator, quote).
//
// Note that `.can-hide` means `display: none;`
// only on non-experimental themes.
// But TBH we don't care about the other themes too much.
@mixin has-content-above {
  // Has a preceding element
  *:not(.can-hide) + &,
  // Has a parent that has a preceding element
  .msg-container *:not(.can-hide) + * & {
    @content;
  }
}
```
and then just

```
@include has-content-above {
    margin-top: 4px;
    border-top-left-radius: 0px;
    border-top-right-radius: 0px;
  }
  ```

Reply:
Very nice, I didn't know of this syntax! I'll rewrite it.